scripts/if_changed.sh

```diff
@@ -50,11 +50,11 @@ else
       ;;
 
     Firestore-xcodebuild)
-      check_changes '^Firestore/(core|third_party)'
+      check_changes '^Firestore'
       ;;
 
     Firestore-cmake)
-      check_changes '^Firestore'
+      check_changes '^Firestore/(core|third_party)'
```

Comment:
FWIW, couldn't we break our cmake build with changes to Firestore/Source obj-c files too?  It's not obvious to me why these aren't both '^Firestore'

Reply:
I don't think cmake currently pulls in the obj-c sources. Check out Firestore/CMakeLists.txt

Reply:
Oh, right.  I was thinking it had to since our C++ code depends on our Obj-C code, but that's all `#if defined(__OBJC__)` away.  So nevermind.  Carry on.

Reply:
Although now that I think about it, perhaps we should add `Protos` to this list?